server/game/cards/12-2T2D/108GraciousGifts.js

```diff
@@ -0,0 +1,37 @@
+const GameActions = require('../../GameActions/index.js');
+const OutfitCard = require('../../outfitcard.js');
+
+class GraciousGifts extends OutfitCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: () => this.game.currentPhase === 'sundown' && (this.owner.getSpendableGhostRock() < this.owner.opponent.getSpendableGhostRock()),
```

Comment:
There is no `opponent` attribute for player, you need to use `getOpponent()` function. Also, the `modifySundownDiscard` effect targets player, so you need to put a `match: this.owner`.
Last thing is that I am not sure about the `getSpendableGhostRock` as it returns GR also from other potential GR sources. I am not sure if the 108 Gracious gifts refers only to player stash, or his overall GR pool.